### PR TITLE
'simple pipeline' notebook update to reflect SDK changes and KF notebook image changes

### PR DESCRIPTION
**Pull request by amygdala**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/705)

### CODE REVIEW

**Comment by amygdala** [comment]
Alexey, would you have time to review this?  It updates from build_python_component to func_to_container_op, plus also changes the command to install/update the kfp sdk. (+ a few other small cosmetic fixes). @Ark-kun

File: pipelines/simple-notebook-pipeline/Simple Notebook Pipeline.ipynb [hunk 1]
```diff
@@ -213,7 +170,7 @@
    "source": [
     "#### Create an Experiment in the Pipeline System\n",
     "\n",
-    "Pipeline system requires an \"Experiment\" to group pipeline runs. You can create a new experiment, or call client.list_experiments() to get existing ones."
+    "Kubeflow Pipelines lets you group pipeline runs by *Experiments*. You can create a new experiment, or call `client.list_experiments()` to see existing ones."
```
Comment (Ark-kun) [inline pipelines/simple-notebook-pipeline/Simple Notebook Pipeline.ipynb]: Let's replace create_experiment+compile+run_pipeline with `client.create_run_from_pipeline_func`, like in the `kubeflow/pipelines`' samples.

**Comment by amygdala** [comment]
What do you think of this version?  I added the 'create_run_from_pipeline_func' example and showed it first, but kept the other variant as an alternative, as people might also find it useful to know how to generate the compiled pipeline artifact. @Ark-kun

**Comment by amygdala** [comment]
just a bump on this :) @Ark-kun

**Comment by cfregly** [comment]
Note the change from Python 3.7.x to 3.6.x.  Not sure this was intentional.  Please verify.

**Comment by amygdala** [comment]
/approve

hi Chris, I believe the key thing is that it's specifying python 3 (vs py27), and will grab any py3 kernel that is available (which will work fine)... so I don't believe that change matters. LMK if you know differently, though.

**Pull request merged**